js/bootstrap-datepicker.js

```diff
@@ -895,6 +895,7 @@
 			if (this.picker.is(':not(:visible)')){
 				if (e.keyCode == 27) // allow escape to hide and re-show picker
```

Comment:
I think you should add `{ .. } ` for the if statement as there are now 2 statements for this statement